crates/services/consensus_module/poa/src/service.rs

```diff
@@ -517,12 +494,13 @@ where
 
         match self.trigger {
             Trigger::Never | Trigger::Instant => {}
-            Trigger::Interval { block_time } => {
-                self.timer
-                    .set_timeout(block_time, OnConflict::Overwrite)
-                    .await;
+            Trigger::Interval { .. } => {
+                return Ok(Self {
+                    last_block_created: Instant::now(),
+                    ..self
+                })
             }
-        };
+        }
```

Comment:
I think we even can remove this part since we have initialization for the `last_block_created` in the constructor=) 
<img width="944" alt="image" src="https://github.com/user-attachments/assets/a410ba49-bd61-4091-a085-c9486c9ecd48">

Reply:
It is not an issue that the constructor can be created long time ago before we make it into_task ?

Reply:
I think it is okay, since the first block after restart could be produced immediately

Reply:
Ok done :)